src/tests/ftest/rebuild/with_ior.py

```diff
@@ -31,17 +27,27 @@ def test_rebuild_with_ior(self):
           -- single pool, single client performing continuous read/write/verify
              sequence while failure/rebuild is triggered in another process
 
-        :avocado: tags=all,daily_regression,small,pool,rebuild,rebuildwithior
+        :avocado: tags=all,daily_regression
+        :avocado: tags=large
+        :avocado: tags=pool,rebuild
+        :avocado: tags=rebuildwithior
+
         """
         # set params
         targets = self.params.get("targets", "/run/server_config/*")
-        rank = self.params.get("rank_to_kill", "/run/testparams/*")
-
-        # ior parameters
-        iorflags_write = self.params.get("F", '/run/ior/iorflags/write/')
-        iorflags_read = self.params.get("F", '/run/ior/iorflags/read/')
-        file1 = "daos:testFile1"
-        file2 = "daos:testFile2"
```

Comment:
I would avoid hardcoding the testfiles under python script. Please add them to YAML file. Assume that you need to enhance this to several ior thread calls in future, this is not the correct way. Based on number of ior calls, you can create necessary testfiles.

Reply:
Thank you for your comment.
Hard-coded file1 and file2 were removed files, current rebuild-ior_utils.py test is using single default test file with daos:/testfile, it can be expended with multiple ior calls to use test.yaml, by util/ior_utils.py and ior_test_base.py.